psydac/core/bsplines.py

```diff
@@ -220,7 +224,8 @@ def basis_funs_1st_der(knots, degree, x, span, out=None):
     ----------
     .. [2] SELALIB, Semi-Lagrangian Library. http://selalib.gforge.inria.fr
     """
-
+    knots = np.asarray(knots)
```

Comment:
As above

Reply:
Done in https://github.com/pyccel/psydac/commit/54fa1fb4c2afba79c77392308a193bad59161fee